locales/PT_br.json

```
@@ -0,0 +1,139 @@
+			},
+			{
+				"question": "O NNID vai continuar funcionando no Pretendo? ",
+				"answer": "Infelizmente, não. Os NNIDs existentes não funcionarão no Pretendo, pois apenas a Nintendo mantém seus dados de usuário; embora uma migração de NNID para PNID seja teoricamente possível, seria arriscado e exigiria dados confidenciais do usuário que não desejamos manter."
+			},
+			{
+				"question": "Como eu uso o Pretendo?",
+				"answer": "Pretendo atualmente não está em um estado que está pronto para uso público. No entanto, quando estiver pronto, você poderá usar o Pretendo simplesmente executando nosso homebrew patcher em seu console."
+			},
+			{
+				"question": "Saberemos quando <feature/service> estará pronto?",
+				"answer": "Não. Muito serviços do Pretendo estão em desenvolvimento independente (por exemplo, Miiverse pode ser trabalhado por um desenvolvedor enquanto as Contas e Amigos estão sendo trabalhados por outro) e, portanto, não podemos fornecer um ETA geral de quanto tempo isso levará."
+			},
+			{
+				"question": "Pretendo funcionará em Emuladores/Cemu?",
+				"answer": "Pretendo é projetado principalmente para o hardware Wii U e 3DS; no momento, o único emulador para esses consoles com suporte para NN é o Cemu. O Cemu não oferece suporte oficialmente a servidores personalizados, mas ainda deve ser possível usar o Pretendo com o Cemu. <br> O Pretendo atualmente não oferece suporte ao Cemu. "
+			},
+			{
+				"question": "Se eu for banido no Nintendo Network, eu vou continuar banido no Pretendo?",
+				"answer": "Não teremos acesso à Banimentos da Nintendo Network e nem todos os usuários serão banidos do nosso serviço. No entanto, teremos regras a seguir ao usar o serviço e o não cumprimento dessas regras pode resultar em banimento."
+			},
+			{
+				"question": "Pretendo suportará Wii/Switch?",
+				"answer": "O Wii já possui servidores personalizados fornecidos por <a href=\"https://wiimmfi.de/\" target=\"_blank\">Wiimmfi</a>. No momento, não desejamos dar suporte ao Switch, pois ele é pago e completamente diferente da Nintendo Network.
+			},
+			{
+				"question": "Eu preciso de desbloqueio para conectar?",
+				"answer": "Sim, você precisará desbloquear seu dispositivo para se conectar; no entanto, no Wii U você só precisará acessar o Homebrew Launcher (ou seja, Haxchi, Coldboot Haxchi ou mesmo o exploit do navegador da web), com informações sobre como o 3DS se conectará em breve."
+			}
+		]
+	},
+	"credits": {
+		"title": "O Time",
+		"text": "Conheça o time de desenvolvedores por trás do projeto",
+		"people": [
+			{
+				"name": "Jonathan Barrow (RedDucks)",
+				"caption": "Criador do projeto e desenvolvedor chefe",
+				"picture": "https://github.com/redduckss.png",
+				"github": "https://github.com/RedDuckss"
```

change to this. he has a new username
```suggestion
				"name": "Jonathan Barrow (jonbarrow)",
				"caption": "Criador do projeto e desenvolvedor chefe",
				"picture": "https://github.com/jonbarrow.png",
				"github": "https://github.com/jonbarrow"
```